app/amigo-request-create-component/amigo-request-create.component.html

```diff
@@ -0,0 +1,8 @@
+<div class="amigo-request-create">
+    <button fab fab-bottom fab-right fab-fixed
+            secondary
+            class="amigo-request-create-fab"
+            [navPush]="onClickPage">
+        <ion-icon name="person-add"></ion-icon>
+    </button>
+</div>
```

Comment:
FAB button that gets added on bottom of Amigos and Amigos Requests pages.

Clicking it takes user to new amigo form.